src/game.cpp

```diff
@@ -294,7 +336,12 @@ void grow_winning_player(game &g)
 
   const int winner_index = get_winning_player_index(g, first_player_index, second_player_index);
   player& winning_player = g.get_player(winner_index);
-  winning_player.grow();
+  const int loser_index = get_losing_player_index(g, first_player_index, second_player_index);
+  player& losing_player = g.get_player(loser_index);
+  if (!is_passive(winning_player) && !is_passive(losing_player))
```

Comment:
I think passive states shouldn't be caught here; instead if either player is passive, there is no collision (and thus, growing/shrinking won't be called)